src/dbus/server/introspect.xml

```diff
@@ -536,6 +536,11 @@
       <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
     </property>
 
+    <!-- PendingDatasetTlvs: The Thread active dataset tlv in binary form. -->
+    <property name="PendingDatasetTlvs" type="ay" access="readwrite">
+      <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
+    </property>
+
```

Comment:
Just noticed this, I'll change after CI is done in case there is anything I need to address.
```suggestion
    <!-- PendingDatasetTlvs: The Thread pending dataset tlv in binary form. -->
    <property name="PendingDatasetTlvs" type="ay" access="read">
      <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
    </property>

```